src/schema/keywords.ts

```diff
@@ -205,6 +224,32 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         hits,
       };
     },
+    autocompleteKeywords: async (
```

Comment:
didn't we just want to run this one on the new table actually?

Reply:
No, we use our keywords as a seed table for autocomplete